src/test/resources/org/broadinstitute/hellbender/tools/copynumber/utils/combine-segment-breakpoints-with-legacy-header-learning-combined-copy-number.tsv

```diff
@@ -0,0 +1,519 @@
+CONTIG	START	END	name	learning_SAMPLE_0	learning_SAMPLE_1	learning_SAMPLE_2	learning_SAMPLE_3	learning_SAMPLE_4	learning_SAMPLE_5	learning_SAMPLE_6	learning_SAMPLE_7	learning_SAMPLE_8	learning_SAMPLE_9	learning_SAMPLE_10	learning_SAMPLE_11	learning_SAMPLE_12	learning_SAMPLE_13	learning_SAMPLE_14	learning_SAMPLE_15	learning_SAMPLE_16	learning_SAMPLE_17	learning_SAMPLE_18	learning_SAMPLE_19	learning_SAMPLE_20	learning_SAMPLE_21	learning_SAMPLE_22	learning_SAMPLE_23	learning_SAMPLE_24	learning_SAMPLE_25	learning_SAMPLE_26	learning_SAMPLE_27	learning_SAMPLE_28	learning_SAMPLE_29	learning_SAMPLE_30	learning_SAMPLE_31	learning_SAMPLE_32	learning_SAMPLE_33	learning_SAMPLE_34	learning_SAMPLE_35	learning_SAMPLE_36	learning_SAMPLE_37	learning_SAMPLE_38	learning_SAMPLE_39	learning_SAMPLE_40	learning_SAMPLE_41	learning_SAMPLE_42	learning_SAMPLE_43	learning_SAMPLE_44	learning_SAMPLE_45	learning_SAMPLE_46	learning_SAMPLE_47	learning_SAMPLE_48	learning_SAMPLE_49	learning_SAMPLE_50	learning_SAMPLE_51	learning_SAMPLE_52	learning_SAMPLE_53	learning_SAMPLE_54	learning_SAMPLE_55	learning_SAMPLE_56	learning_SAMPLE_57	learning_SAMPLE_58	learning_SAMPLE_59	learning_SAMPLE_60	learning_SAMPLE_61	learning_SAMPLE_62	learning_SAMPLE_63	learning_SAMPLE_64	learning_SAMPLE_65	learning_SAMPLE_66	learning_SAMPLE_67	learning_SAMPLE_68	learning_SAMPLE_69	learning_SAMPLE_70	learning_SAMPLE_71	learning_SAMPLE_72	learning_SAMPLE_73	learning_SAMPLE_74	learning_SAMPLE_75	learning_SAMPLE_76	learning_SAMPLE_77	learning_SAMPLE_78	learning_SAMPLE_79	learning_SAMPLE_80	learning_SAMPLE_81	learning_SAMPLE_82	learning_SAMPLE_83	learning_SAMPLE_84	learning_SAMPLE_85	learning_SAMPLE_86	learning_SAMPLE_87	learning_SAMPLE_88	learning_SAMPLE_89	learning_SAMPLE_90	learning_SAMPLE_91	learning_SAMPLE_92	learning_SAMPLE_93	learning_SAMPLE_94	learning_SAMPLE_95	learning_SAMPLE_96	learning_SAMPLE_97	learning_SAMPLE_98	learning_SAMPLE_99
```

Comment:
This test file seems a little on the big side, but I'll leave it up to you if you want to trim it down.